src/hepattn/models/wrapper.py

```diff
@@ -177,3 +178,27 @@ def mlt_opt(self, losses, outputs):
             mtl_backward(losses=layer_losses, features=layer_features, aggregator=UPGrad())
 
         opt.step()
+
+    def log_figure(self, name, fig, step=None):
```

Comment:
As far as I know everyone is using comet, can we remove some of the structure here and just call `log_image` directly from the callback? I don't think `wrapper.py` should import `matplotlib`, but it's fine if the callback does